mart/configs/callbacks/adversary_connector.yaml

```diff
@@ -4,3 +4,4 @@ adversary_connector:
   train_adversary: null
   val_adversary: null
   test_adversary: null
+  batch_c15n: ???
```

Comment:
It would be nice if the default was `tuple`? Or whatever default works well with torchvision models. I'm still confused about the difference between `tuple` and `list` and for some reason can't keep them straight.

Reply:
Good idea. Set a `tuple` batch_c15n by default should work for most experiments in MART, because datamodules often give a batch `(input, target)` or `[input, target]`.

Fixed in 3192a99986360fe66bf02894296ff34c61872da4